### DISCUSSION TITLE
Unable to login into NC after executing migration.md steps

### DISCUSSION THREAD

**Discussion by dtretyakov** [nextcloud/all-in-one#1679] [Questions] [answered]
### Steps to reproduce
1. Follow migration.md steps for "Migrate the files and the database"
2. Try to login by migrated user

### Expected behavior
Existing user could login with their credentials

### Actual behavior
Unable to login

### Host OS
Alpine

#### Nextcloud AIO version
Nextcloud AIO v4.0.1

#### Current channel
stable

#### Other valuable info

I tried to follow migration twice with the same result.

According to the container logs database dump was imported.

Also, it's impossible to login by using original admin account with password stored in the `/var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data/data/configuration.json` file.

**Comment by dtretyakov** [comment]
So there is no way to execute last step:
> 13. As last step, install all apps again that were installed before on your old instance by using the webinterface.

**Comment by dtretyakov** [comment, marked as answer]
> docker exec --user www-data -it nextcloud-aio-nextcloud php occ user:resetpassword <username>

solved that problem.

**Comment by szaimen** [comment]
Moving to discussions so that other people can find your solution
